### PR TITLE
Pacman move

### PR DESCRIPTION
**Pull request by mcavo**
Add Pacman movement and ghost model

Solves #18 and #22

### CODE REVIEW

File: tp2/Pacmania/Assets/Scripts/GameLoader.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+	public GameObject gameManager;			//GameManager prefab to instantiate.
+	//public GameObject soundManager;			//SoundManager prefab to instantiate.
+
+
+	void Awake ()
+	{
+		//Check if a GameManager has already been assigned to static variable GameManager.instance or if it's still null
+		if (GameManager.instance == null)
+			//Instantiate gameManager prefab
```
Comment (eugis) [inline tp2/Pacmania/Assets/Scripts/GameLoader.cs:14]: As the rest of the code has `{ ... }` in the if blocks, we could add here o remove the others. 
Personally I prefer the first option.

File: tp2/Pacmania/Assets/Scripts/GameManager.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	
+	}
+
+	void InitGame() {
+		score = 0;
```
Comment (eugis) [inline tp2/Pacmania/Assets/Scripts/GameManager.cs:43]: We have twice the initialization of `score = 0` shouldn't be only one ?

**Review by eugis — COMMENTED** [review]
It's great @mcavo !!

File: tp2/Pacmania/Assets/Scripts/GameLoader.cs (hunk 1 above)
Comment (mcavo) [reply tp2/Pacmania/Assets/Scripts/GameLoader.cs:14]: Done!

File: tp2/Pacmania/Assets/Scripts/GameManager.cs (hunk 2 above)
Comment (mcavo) [reply tp2/Pacmania/Assets/Scripts/GameManager.cs:43]: Done! I let the one inside InitGeme, and remove the definition inside the Start function.

We have to discuss this later, if we implements levels maybe this wouldnt be the place to define the score and the lives.

**Pull request merged**